package.json

```diff
@@ -51,6 +51,7 @@
     "react-native-render-html": "^4.1.2",
     "react-native-sentry": "^0.42.0",
     "react-native-share": "^1.2.1",
+    "react-native-simple-toast": "^1.0.0",
```

Comment:
I think we should not pull in native dependencies for hidden development features.

* The iOS native dependency wasn't updated in the last 2 years.
* I'm sure we can create a js-only solution (like https://www.npmjs.com/package/react-native-easy-toast)

If we do not want to do it ourself, there is also a js-only library with more downloads: https://www.npmjs.com/package/react-native-easy-toast

I suggest to use the js-only library (as it will make upgrading in 1-2 months easier) and create a task to replace it with a custom version if required.

Reply:
After we now also display the overriding API-URL and a "Switch back" button I think that the toast functionality is no longer necessary. I remove the package and only display it on Android now.